app-emulation/fuse/metadata.xml

```diff
@@ -17,5 +17,6 @@
 		<flag name="backend-sdl">Use SDL rendering backend</flag>
 		<flag name="backend-svga">Use svgalib rendering backend</flag>
 		<flag name="backend-X">Use X11 rendering backend</flag>
+		<flag name="memlimit">Add memory usage limiting</flag>
```

Comment:
I confirm that it lowers compile-time memory usage of z80_ops.c from 263 MiB to 140 MiB with gcc 7.3.0, and from 137 MiB to 117 MiB with clang-7.0.0. Since it applies to compile-time only (not to runtime memory usage), I suggest modifying it to "Add compile-time memory usage limiting".

Reply:
Hmm, sounds like I did the same thing in pypy as `low-memory`. `memlimit` makes me think more of allowing user to set a limit on memory usage.